send_rx_functions.php

```diff
@@ -373,3 +373,49 @@ function send_rx_get_user_pharmacies($project_id, $username = USERID, $project_t
 
     return $pharmacies;
 }
+
+function send_rx_get_pharmacy_users($project_id, $pharmacy_id, $role = null, $project_type = 'patient') {
+    if($project_type == 'patient'){
```

Comment:
Let's adopt some sort of PHP coding standards and add parenthesis between `if` and `(`, and between `)` and `{`. It also applies for another lines below.